### DISCUSSION TITLE
Match-Case functionality

### DISCUSSION THREAD

**Discussion by macjones13** [QuirkyCort/gears#106] [Q&A]
With the new addition of Match-Case to python, is it possible to use it in GearsBot?  If so how is it implemented?  I've tried to use it with no luck.  I used to use Not Exactly C (NXC) in my robotics classes and showed my students how to use Switch-Case.  I'd love to be able to do that again.  Any help would be welcomed.

Thank you,

Mac Jones

**Comment by QuirkyCort** [comment]
Gears uses the Skulpt Python interpreter, so not everything that's in cPython are available. Match-Case is pretty new, so it'll likely be a long time before it trickle down to Skulpt.

**Reply by macjones13** [reply]
Thank you for the info.  I hope Skulpt picks it up sooner than later.  It is nice to see that python decided to add a switch analogue.  I'll look forward to using it once it's available. 

Thanks again,

Mac Jones
